AppCenter/AppCenter/Internals/Sender/MSHttpSender.m

```diff
@@ -254,19 +254,10 @@ - (void)sendCallAsync:(MSSenderCall *)call {
   }
 }
 
-- (void)call:(MSSenderCall *)call completedWithFatalError:(BOOL)fatalError {
+- (void)call:(MSSenderCall *)call completedWithResult:(MSSenderCallResult)result {
   @synchronized(self) {
-    NSString *callId = call.callId;
```

Comment:
The deleted block was executed even for fatal error but your change doesn't. Any intentions?

Reply:
The deleted call gets executed for all calls but at the end of this method. See [here](https://github.com/Microsoft/AppCenter-SDK-Apple/pull/891/files/ffee0191010ee051b126548a49726e26a518a61f#diff-7de131038a5e31d3fa3c7bf3557d1ef7R281).